docs/getting-started/advanced/integrating-from-source-webpack.md

```diff
@@ -93,7 +93,7 @@ module.exports = {
 	module: {
 		rules: [
 			{
-				test: /ckeditor5-[^/\\]+[/\\]theme[/\\]icons[/\\][^/\\]+\.svg$/,
+				test: /ckeditor5-icons\/theme\/icons\/[^/\\]+\.svg$/,
```

Comment:
It looks like there are more problems with this regexp than expected 🥹 

Previous regexp with `[^/\\]` is invalid - unescaped `/` delimiter.
Current regexp also contains the same invalid `[^/\\]` fragment.

Also, for some reasons we were defining these regexps with both `\` and `/` path separators. Not sure if we can just support unix-style `/` now.

Reply:
> Previous regexp with `[^/\\]` is invalid - unescaped `/` delimiter.

Not true, sorry. My mistake, because when testing this regexp I chose PCRE2, not JS flavor. The regexp fragment `[^/\\]` is valid in JS.

So question is only whether we want to use only unix-style `/` path separator.

Reply:
How about we just go with `/\.svg$/`? I don't expect it to cause any more issues compared to the current version.

Reply:
Yes, good idea.

Reply:
Updated